docs/security/10-02-forbidden-console.md

```diff
@@ -0,0 +1,18 @@
+---
+title: '"403 Forbidden" in the Console'
+type: Troubleshooting
+---
+
+If you log to the Console and get the "403 Forbidden" error, do the following:
```

Comment:
```suggestion
If you log to the Console and get the `403 Forbidden` error, do the following:
```

Reply:
Sorry, I changed it now to unify it with the formatting of other errors in the docs.